bindings/pybind11/CMakeLists.txt

```diff
@@ -19,7 +43,7 @@ set_target_properties(pybind11_idyntree PROPERTIES
 
 # if compile tests execute also python tests
 if(IDYNTREE_COMPILE_TESTS)
-    add_subdirectory(tests)
+    add_subdirectory(tests idyntree_pybind11_test)
```

Comment:
Do you think there is a way to avoid this and avoid modifying the test Cmake working directory variable?

Reply:
@GiulioRomualdi any thoughts on this? Note that if it is difficult I prefer to merge the PR in the current form, it is not particularly problematic to have this.